docs/source/metrics.rst

```diff
@@ -381,6 +381,56 @@ Threshold Brier Score
 
 .. autofunction:: _threshold_brier_score
 
+Ranked Probability Score
+========================
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['rps']}")
+
+.. autofunction:: _rps
+
+Reliability
+===========
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['reliability']}")
+
+.. autofunction:: _reliability
+
+Discrimination
+==============
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['discrimination']}")
+
+.. autofunction:: _discrimination
+
+Rank Histogram
+==============
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['rank_histogram']}")
+
+.. autofunction:: _rank_histogram
```

Comment:
How do I get this new function documented? Not listed under api. Yet? #438 ?

Reply:
It looks like this is working on my end on the metrics page. Will add it to global API with #438.